config/300-clusterimagepolicy.yaml

```diff
@@ -99,6 +129,9 @@ spec:
                                   type: string
                           url:
                             type: string
+                      name:
+                        description: Name is the name for this authority. Used by the CIP Policy validator to be able to reference matching signature or attestation verifications. If not specified, the name will be attestation-<index in array>
```

Comment:
`attestation-%d`?  Why not `authority-%d`?

Reply:
I failed to update comment, I originally had it as attestation, and realized it was not good so I changed it to authority-%d, but forgot to update the comment.